### DISCUSSION TITLE
buddhism/2023/heart_sutra/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#85] [General]
# buddhism/2023/heart_sutra/

https://shanenull.com/buddhism/2023/heart_sutra/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/skhandas/
https://shanenull.com/buddhism/2023/sense/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/heart_sutra/
